features/admin.actions.v1/package.json

```
@@ -0,0 +1,35 @@
+{
+    "private": true,
+    "name": "@wso2is/admin.actions.v1",
+    "version": "1.0.0",
```

```suggestion
    "version": "0.0.0",
```

Keep this at `1.0.0` and add a major changeset when you're ready to merge so it'll release a `1.0.0`

---

Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/d5b3f7c8d61759691a8fd6e271fbb145d39b68d3